src/commands/synthetics/utils/public.ts

```diff
@@ -507,11 +507,12 @@ export const getTestsToTrigger = async (
   const errorMessages: string[] = []
   // When too many tests are triggered, if fetched from a search query: simply trim them and show a warning,
   // otherwise: retrieve them and fail later if still exceeding without skipped/missing tests.
-  if (triggerConfigs.length > MAX_TESTS_TO_TRIGGER && triggerFromSearch) {
+  if (triggerFromSearch && triggerConfigs.length > MAX_TESTS_TO_TRIGGER) {
```

Comment:
Doesn't it mean that if more than 100 tests are triggered from test files, it won't show the warning, and more importantly, it'll trigger all of the tests?

Reply:
No. The comment in the line above says that if more than 100 tests are triggered _from test files_, we will throw an error:

https://github.com/DataDog/datadog-ci/blob/aa2aca22eb480f9c0d045c8a4724bc59ed7451af/src/commands/synthetics/utils/public.ts#L579-L584